scripts/ninja.js

```diff
@@ -585,13 +585,33 @@ function sourceToTarget(y) {
  */
 function ocamlDepForBscAsync(files, dir, depsMap) {
   return new Promise((resolve, reject) => {
+    let mlfiles = [];
+    files.forEach(f => {
+      if (f.endsWith(".res") || f.endsWith(".resi")) {
+        isIntf = f.endsWith(".resi");
+        let mlname = f.slice(0, isIntf ? -5 : -4) + (isIntf ? ".mli" : ".ml");
+        mlfiles.push(mlname);
+        try {
+          cp.execSync(`${bsc_exe} -dsource ${f} 2>${mlname}`, {
```

Comment:
Unfortunately this also builds, not only converts to `.ml`, which overwrites the `.cmj` file and creates problems when one builds the same file again.
Some dedicated option to only do the conversion, and nothing else, needs to be provided.